setup.sh

```diff
@@ -298,16 +298,28 @@ function synchronize_files()
   setup_config_file
   ASSERT_IF_NOT_EQ_ZERO "Config file failed" "$?"
 
-  if [ -f "$HOME/.bashrc" ]; then
-      # Add to bashrc
-      echo "# $app_name" >> "$HOME/.bashrc"
-      echo "source $libdir/$BASH_AUTOCOMPLETE.sh" >> "$HOME/.bashrc"
+  if [[ -x "$(command -v bash)" ]]; then
```

Comment:
btw, @magalilemes added a helper named `command_exists()` (latest code in the unstable branch); please use that helper in your new PR.